lib/flutter_typeahead.dart

```diff
@@ -1349,6 +1349,7 @@ class _SuggestionsBoxController {
   static const int waitMetricsTimeoutMillis = 1000;
 
   final BuildContext context;
+  final FocusNode focusNode;
```

Comment:
I think FocusNodes need to be disposed of in the dispose() method .. ??

Reply:
Good catch. Fixed in new commit. It is now properly disposed and I added a safety check in case the widget is disposed before it reaches that point.